tests/Unit/HelperMethodTest.php

```diff
@@ -96,4 +99,46 @@ public static function userTimeDataProvider(): array {
             [$illuminateCarbonDefaultTime, 'H:i', false],
         ];
     }
+
+    /**
+     * @dataProvider stationBoardTimezoneOffsetProvider
+     */
+    public function testStationBoardTimezoneOffset($expected, $departures): void {
+        $this->user = User::factory()->make();
+        Passport::actingAs($this->user);
+
+        $user = Mockery::mock($this->user)
+                       ->shouldReceive('getAttribute')
+                       ->with('timezone')
+                       ->andReturn('Europe/Berlin')
```

Comment:
Does that break in 3 weeks? `Europe/Berlin` does not explicitly say if its Daylight Savings Time, so I'm a little worried if the test will fail then.

Reply:
Oh, yes. You're definitely right.

I'll test it locally (+02:00 at that date **should** always compute to Europe/Berlin, but I'll verify it soon)

Reply:
This comment saved me a lot of future headaches! Thanks you!

Technically, this still would've worked but only if you didn't search for the previous day (you're in CET and yesterday is in CEST) - now it'll always work.

Also added test cases for CET/CEST.